Bio/SeqFeature.py

```diff
@@ -851,6 +851,16 @@ def __iter__(self):
             for i in range(self._start, self._end):
                 yield i
 
+    def __eq__(self, other):
+        """Check if start, end and strand of this location are the same as in the other location"""
+        if not isinstance(other, FeatureLocation):
+            return False
+        return self._start == other.start and self._end == other.end and self._strand == other.strand
```

Comment:
We need to check ``.ref`` and ``.ref_db`` as well, which come up in things like assembly contig entries or trans-splicing. That would need a test too.

After which the docstring can be simplified to something like:

```python
"""Implement equality by comparing all the location attributes."""
```